MovieAppPetProject/MovieAppPetProject/HomeScreen/NewTableViewCell.swift

```diff
@@ -51,4 +51,9 @@ class NewTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionV
         }
         return cell
     }
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let selectedMovie = nowPlayingMovies[indexPath.item]
+        parentViewController?.performSegue(withIdentifier: "goToMovieDetails", sender: selectedMovie)
```

Comment:
make a static let for the identifier and don't use the string "goToMovieDetails" again😄